src/core/map/coord.rs

```diff
@@ -504,36 +507,36 @@ mod coord_tests {
 				around
 			);
 		}
-	}
+	);
 
 	#[test]
-	fn coord_orientation_neighbor_iterator_count() {
+	fn coord_orientation_neighbor_iterator_small_count() {
 		{
```

Comment:
This block can probably be flattened since this is the only part in this function that remains.